api/src/resources/resourcesApi.ts

```diff
@@ -60,4 +60,14 @@ export interface ResourcesApi {
      * @param options - Options for revealing the resource. See {@link vscode.TreeView.reveal}
      */
     revealAzureResource(id: string, options?: VSCodeRevealOptions): Promise<void>;
+
+    /**
+     * Gets a list of Azure resources (ARM ID and maybe kind) that have been recently used/interacted with in the Azure Resources extension.
+     */
+    getRecentlyUsedAzureResources(): Promise<{ id: string, kind?: string }[]>;
```

Comment:
I don't have a hard feeling on the names of these functions. Please tell me if you want them changed.

Reply:
Should we make a note that it doesn't track subscriptions or resource groups?